app/views/articles/_full_comment_area.html.erb

```diff
@@ -2,10 +2,28 @@
   <section id="comments" data-follow-button-container="true" data-updated-at="<%= Time.current %>" class="text-padding mb-4 border-t-1 border-0 border-solid border-base-10">
     <% if @article.show_comments %>
       <header class="relative flex justify-between items-center mb-6">
-        <h2 class="crayons-subtitle-1">
-          <%= t("views.articles.comments.subtitle_html",
+        <h2 class="crayons-subtitle-1" id="toggle-comments-sort-dropdown" aria-controls="comments-sort-dropdown" aria-expanded="false" aria-label="Toggle Comment Sort Order dropdown menu" aria-haspopup="true">
+          <%= t("views.articles.comments.subtitle.#{@comments_order}_html",
                 num: tag.span(t("views.articles.comments.num", num: @article.comments_count), class: "js-comments-count", data: { comments_count: @article.comments_count })) %>
         </h2>
+        <div id="comments-sort-dropdown">
+          <h4 class="comments-sort-title">Sort discussion: </h4>
+render "comments/sort_option",
```

Comment:
Refactor (blocking): There are missing `<%=` for rendering partials.

```suggestion
<%= render "comments/sort_option",
```

![CleanShot 2022-03-07 at 10 27 22](https://user-images.githubusercontent.com/833231/157064219-de415731-8102-4319-927b-8f074e0199f0.png)

Reply:
thats really odd, must be some auto-fix while committing. Will fix this